src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIllegalTypeTest.java

```diff
@@ -63,7 +63,7 @@ public void testOne() throws Exception {
     @Test
     public void testTwo() throws Exception {
```

Comment:
Do you plan to fix such names to match input suffix?

Reply:
Or better to do this in separate PR ? I think so

Reply:
@romani , in fact I've been changing respective test method names too in these PRs itself..

The problem with a few input files is that there is no proper design on how they have been introduced to our code by the authors. I'm unable to come up with meaningful suffixes for such files, and am leaving those abstract (One, Two, ....)

Rest assured, the input file suffixes and test names, I always make sure they match.
Even here, `testTwo` because it takes file `InputXpathIllegalTypeTwo`

Reply:
Ok, thanks a lot for details